pallets/communities/src/lib.rs

```diff
@@ -393,6 +404,11 @@ pub mod pallet {
 		///
 		/// [11]: `types::CommunityMetadata`
 		#[pallet::call_index(1)]
+		#[pallet::weight(<T as Config>::WeightInfo::set_metadata(
```

Comment:
Because it's the only one that doesn't directly respond to a _standard_ weight, and instead, is parametrized by on the length of each input.

Other weights are automatically associated to their corresponding call via [this macro](https://github.com/virto-network/virto-node/pull/340/files/169ddfdf08798417e3f358bee86e4288f7f25e90#diff-68ab14e4e73e9d5ab860d9abf84ceebc01ad8cd0e5dd0b5ff04baf262ab69e57R383) at the begining of the call impl.